apmpackage/apm/0.1.0/manifest.yml

```diff
@@ -62,6 +62,12 @@ policy_templates:
         required: false
         description: Enter as <Id>:<API Key>
         show_user: true
+      - name: default_service_environment
```

Comment:
not sure about this, shouldn't we use the `namespace` instead?

Reply:
I'd prefer to keep them separate for now, so users have the flexibility to have two dimensions:
 - a namespace is a collection of APM services, e.g. split by business applications, business unit, etc.
 - typical environments for those services: production, testing, development

Reply:
ah ok, that was news to me 👍

Reply:
Please also update the [README](https://github.com/elastic/apm-server/tree/master/apmpackage/apm/0.1.0/docs#namespace) accordingly, it refers to using `namespace` for environment.

Reply:
Just to be clear, setting namespace to environment is still a reasonable choice. Perhaps we should just soften the language a little, to point it out as an option but not necessarily state that it is recommended.

Reply:
yeah, that's what I meant, the additional use case for `namespace` tha you explained above, should be added to the README.

Reply:
done